xarray/tutorial.py

```diff
@@ -226,15 +226,24 @@ def load_dataset(*args, **kwargs):
         return ds.load()
 
 
-def scatter_example_dataset():
+def scatter_example_dataset(*, seed=None) -> Dataset:
+    """
+    Create an example dataset.
+
+    Parameters
+    ----------
+    seed : integer, optional
+        Seed for the random number generation. The default is None.
+    """
+    rng = np.random.default_rng(seed)
     A = DataArray(
         np.zeros([3, 11, 4, 4]),
         dims=["x", "y", "z", "w"],
         coords=[
-            np.arange(3),
-            np.linspace(0, 1, 11),
-            np.arange(4),
-            0.1 * np.random.randn(4),
+            ("x", np.arange(3)),
+            ("y", np.linspace(0, 1, 11)),
+            ("z", np.arange(4)),
+            ("w", 0.1 * rng.standard_normal(4)),
```

Comment:
it looks like this `list` of key-value `tuple`s should just become a `dict`...

Reply:
Done.